scheduler/scheduler.go

```diff
@@ -250,6 +276,7 @@ func BuildDependencyGraph(c client.Interface, sel labels.Selector) (DependencyGr
 			depGraph[child].Requires, depGraph[parent])
 
 		depGraph[child].Meta[parent] = d.Meta
+		depGraph[parent].Meta[parent] = d.Meta
```

Comment:
We discussed it already. If you look at the way get-status is iterating over the tree, you will see that I need the metadata to be accessible on parent to be able to check the parent's status. Do you want me to put comments there?